static/js/stat_var_hierarchy/stat_var_section_input.tsx

```diff
@@ -34,6 +34,30 @@ import { USA_CHILD_PLACE_TYPES } from "../tools/map/util";
 
 const TOOLTIP_TOP_OFFSET = 10;
 const TOOLTIP_RIGHT_MARGIN = 20;
+const PLACE_TYPE_MAPPING = {
+  EurostatNUTS1: "State",
+  EurostatNUTS2: "State",
+  Town: "City",
+  Village: "City",
+  Borough: "City",
+};
+const IGNORED_PLACE_TYPES = new Set([
+  "Continent",
+  "CommutingZone",
+  "PowerPlantUnit",
+  "PoliticalCampaignCmte",
+  "AirQualitySite",
+  "CongressionalDistrict",
+  "ElementarySchoolDistrict",
```

Comment:
Ok for now, but I forget what we finally decided on school districts and tracts... to show on a toggle? Or to show when we can embed links in the text?

Also, maybe we want to add CensusBlockGroup to this list.

Reply:
what is the toggle? I think until we have links, we shouldn't include places that can't be searched for.